src/services/twilio-api/get-help-doc-link.js

```diff
@@ -0,0 +1,64 @@
+const jsonMap = require('../api-doc-mapping.json');
+// json structure = url: {path : {topics: {}, commands: {}}, subpath : {wherever applicable}}
+const urlMap = new Map(Object.entries(jsonMap));
+
+const getSubroute = (cmdDetailsArr, subpathMap) => {
+  const subdomain = cmdDetailsArr[3];
+  // apiName/version to locate sub url in json map
+  const apiVersion = `${cmdDetailsArr[1]}/${cmdDetailsArr[2]}`;
+  const subUrlArr = subpathMap.get(apiVersion);
+  if (subUrlArr === undefined) {
+    return '';
+  }
+  const subUrlMap = new Map(Object.entries(subUrlArr));
+  const subUrl = subUrlMap.get(subdomain);
+  return subUrl;
+};
+
+const getRootPath = (cmdDetailsArr, subpathMap, topicsMap, commandsMap) => {
+  const topicName = cmdDetailsArr[0];
+  const apiName = cmdDetailsArr[1];
+  let url = '';
+
+  if (topicName === 'api') {
+    // get the array of apis and convert to a map
+    const arrOfAllApis = topicsMap.get(topicName);
+    if (arrOfAllApis === null) {
```

Comment:
[Q] Instead of checking specifically for `null` Can't we use `if (!arrOfAllApis) { return ''; }`.

Reply:
'!' checks for 0, NaN, null, undefined everything, that's why I haven't used it predominately. In this case, this is just a sanity check before creating a map out of the array, so '!' would be fine. I'll make the change :)